vaadin-select-flow-parent/vaadin-select-flow/src/main/java/com/vaadin/flow/component/select/Select.java

```diff
@@ -1045,4 +1065,45 @@ private void identifierProviderChanged(
         keyMapper.setIdentifierGetter(identifierProvider);
     }
 
+    @Override
+    protected void validate() {
+        boolean isRequired = this.isRequiredIndicatorVisible();
```

Comment:
**nit:** Could use `ValidationUtil.checkRequired` for the sake of consistency.

```suggestion
        ValidationResult requiredValidation = ValidationUtil.checkRequired(isRequiredIndicatorVisible(), getValue(), getEmptyValue());
        setInvalid(requiredValidation.isError());
```

Reply:
Done